### DISCUSSION TITLE
can I create pop ups (longer text descriptions) when hovering over a column header?

### DISCUSSION THREAD

**Discussion by markwheal** [fbaligand/kibana-enhanced-table#241] [Q&A]
I've not tried, but would the "apply templates" allow me to do such a thing?

**Comment by fbaligand** [comment]
Well, templates allow you to generate html. 
so you can use “title” html attribute to get hover text or even css to have something more beautiful.
https://stackoverflow.com/questions/11022843/add-hover-text-without-javascript-like-we-hover-on-a-users-reputation
